### ISSUE TITLE
error when trying to do incremental deployment

### ISSUE THREAD

**Issue by leandroninja** [Azure-Player/azure.datafactory.devops#173]
After updating the incremental deployment method, I looked at the documentation as you instructed, created the storage, passed the path and gave permission but I get this error.
Can you help me?

[error]Exception calling "Exists" with "0" argument(s): "This request is not authorized to perform this operation using this permission."

**Comment by NowinskiK** [comment]
Apparently your Azure DevOps Agent doesn't have enough permission to Storage's Container.

**Comment by leandroninja** [comment]
yes, it has the permissions, I tried to run it again now it shows another error ##[error]The property 'Deployed' cannot be found on this object. Verify that the property exists.

**Comment by leandroninja** [comment]
Yes, that's a problem, I gave up using the feature that was very important because it reduced our deployment from 50 minutes to 15 minutes, but after the change to save in storage, I couldn't make it work, I tried in countless ways but without success.

**Comment by NowinskiK** [comment]
Can you retest the feature on your side with the latest version 2.3.1778?
I tested it today and it works:

![Image](https://github.com/user-attachments/assets/07a4da1d-2055-4b03-9d69-059378f70615)

![Image](https://github.com/user-attachments/assets/8b18eb3e-57f1-4296-9088-e39388688cad)
